docs/sasl-mechs.md

```diff
@@ -4,8 +4,9 @@ title: SASL Mechanisms
 ---
 IAL primarily uses the same mechanisms as SASL in other protocols. Most commonly:
 
-* [EXTERNAL](https://tools.ietf.org/html/rfc4422#appendix-A) as defined by RFC 4422
 * [PLAIN](https://tools.ietf.org/search/rfc4616) as defined by RFC 4616
+* [EXTERNAL](https://tools.ietf.org/html/rfc4422#appendix-A) as defined by RFC 4422
+* [ECDSA-NIST256P-CHALLENGE](https://github.com/kaniini/ecdsatool)
```

Comment:
But ecdsatool isn't related to the specification or do you see those RFCs including anything else either?